### ISSUE TITLE
ProjectRed Inductive Furnace?

### ISSUE THREAD

**Issue by BoBoLittleStar** [GregTech6/gregtech6#33]
<img width="342" alt="image" src="https://user-images.githubusercontent.com/44244431/107021728-7bf5b400-67df-11eb-943d-43a0a4af81d0.png">
: |

**Comment by BoBoLittleStar** [comment]
I put the black granite in and it still outputs black granite...

**Comment by GregoriusT** [comment]
Wait what the heck happens there, does this happen with vanilla Furnaces too??

**Comment by BoBoLittleStar** [comment]
Vanilla furnaces are good. Been using for a long time

**Comment by BoBoLittleStar** [comment]
Also, this inductive furnace turns wood log into charcoal : D

**Comment by GregoriusT** [comment]
Yeah, that is technically not my responsibility, that Furnace is clearly just broken in on itself. I'm not even interacting with it.

How does it even turn regular Cobblestone into Black Granite?! Like how does it do the Recipes THIS wrong?!
